apps/consulting/pages/_app.tsx

```diff
@@ -12,6 +12,17 @@ function CustomApp({ Component, pageProps }: AppProps): JSX.Element {
       <Meta />
       <Component {...pageProps} />
 
+      <Script
+        strategy="afterInteractive"
+        src={`https://www.googletagmanager.com/gtag/js?id=${process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS}`}
+      />
+      <Script id="google-analytics-script" strategy="afterInteractive">
+        {`window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', '${process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS}');`}
```

Comment:
Could we rewrite this as:

```js
window.dataLayer = window.dataLayer || [];
function gtag(){dataLayer.push(arguments);}
gtag('js', new Date());
gtag('config', process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS);
```

? I don't see the need to quote the entire code snippet.

Reply:
@kherma can probably speak to this better than I can, but when I was trying earlier to implement this myself, whenever it wasn't quoted, Typescript (I think it was) complained vociferously.

Reply:
Oh I see, [it's a Next.js thing](https://nextjs.org/docs/basic-features/script#inline-scripts).